docs/hub-deployment-guide/configure-auth/dummy-auth.md

```diff
@@ -0,0 +1,110 @@
+# Dummy Authentication
```

Comment:
Let's call this 'shared password authentication', and keep 'Dummy authenticator' as an implementation detail. We should not set this up without a shared password.